src/encoded/static/components/search.js

```diff
@@ -696,9 +713,24 @@ const DatasetComponent = (props, reactContext) => {
             <div className="result-item">
                 <div className="result-item__data">
                     <a href={result['@id']} className="result-item__link">
-                        {datasetTypes[result['@type'][0]]}
+                        {fccSeries ? '' : <span>{`${datasetTypes[result['@type'][0]]}`}</span>}
                         {seriesDataset ?
                             <span>
+                                {(perturbationType && perturbationType.length > 0 && fccSeries) ?
```

Comment:
Generally you should check the easy cases first, like if `fccSeries` is true, before more complicated things like the length of `perturbationType`. That way if `fccSeries` is false, JS just moves on without checking the length of `perturbationType`. In this case, it has nearly no performance penalty.